docs/src/Installation.md

```diff
@@ -0,0 +1,74 @@
+# Installing CLIMA
+
+Installation of CLIMA can be divided into 3 parts:
+
+1. Installation of Julia (1.3.1)
+2. Installation of MPI
+3. Installation of CLIMA
+
+## Installation of Julia (1.3.1)
+
+The Climate Machine (CLIMA) uses the Julia programming language; however, at the current time, certain dependencies are only available on Julia 1.3, so please download the Julia source from [Julia's old releases](https://julialang.org/downloads/oldreleases/#v131_dec_30_2019).
```

Comment:
'so please download the Julia source from' => 'so please download the appropriate binary for your platform from'